react-native/react/engine/index.js

```diff
@@ -73,7 +69,43 @@ class Engine {
     const callMap = this.sessionIDToIncomingCall[sessionID]
 
     if (callMap && callMap[method]) {
-      callMap[method](param, response)
+      // make wrapper so we only call this once
+      let once = false
```

Comment:
This once stuff is to only allow you to call an rpc callback once. instead of having this logic everywhere its in this wrapped response object. we might want to nil out the response objects when we call them to reduce memory load and make GC easier...